### ISSUE TITLE
With talkback on, unable to click on the Concept card in Revision tab

### ISSUE THREAD

**Issue by KolliAnitha** [oppia/oppia-android#4368]
**Describe the bug**
With talkback on, unable to click on the Concept card in Revision tab

**To Reproduce**
1. Go to 'Addition and Subtraction'
2. Click on Revision tab
3. Double tap on 'Estimation'
4. With talkback on, try to click on the concept card link
5. Unable to select the concept card link

**Expected behavior**
With talkback on the user should be able to click on the concept card and open it

https://user-images.githubusercontent.com/101634267/169784671-18139373-3dbe-4c09-8b48-9ea96d6ab72d.mp4

Environment:
Device name: One plus Nord2 5G
Android version : Android 11
App version: 0.7-alpha-6c08d882d3

**Comment by BenHenning** [comment]
It's my understanding that normally one would open the link context menu for this sort of thing. See the 'Testing' section at https://support.google.com/accessibility/android/answer/6378148?hl=en. Does this work as documented @KolliAnitha or is the link completely inaccessible?

**Comment by KolliAnitha** [comment]
Yes @BenHenning. It is accessible via the link context menu. We can close the issue

**Comment by BenHenning** [comment]
Sounds good--thanks for confirming @KolliAnitha!
